homeworks/alexander.yatsenko_Syreon514/homework_6/task1.js

```diff
@@ -0,0 +1,35 @@
+function createUser(user) {
+    let value;
```

Comment:
using single protected variable for both assignments, not good

Reply:
Understood. Fixed